docs/index.md

```diff
@@ -23,13 +23,13 @@ terraform {
 
 provider "buildkite" {
   api_token    = "token" # can also be set from env: BUILDKITE_API_TOKEN
-  organization = "slug"  # can also be set from env: BUILDKITE_ORGANIZATION
+  organization = "slug"  # can also be set from env: BUILDKITE_ORGANIZATION_SLUG
 }
 ```
 
 ## Argument Reference
 
-- `api_token` - (Required) This is the Buildkite API Access Token. It must be provided but can also be sourced from the `BUILDKITE_API_TOKEN` environment variable.
-- `organization` - (Required) This is the Buildkite organization slug. It must be provided, but can also be sourced from the `BUILDKITE_ORGANIZATION` environment variable. The token requires GraphQL access and the `write_pipelines, read_pipelines` scopes.
-- `graphql_url` - (Optional) This is the base URL to use for GraphQL requests. It defaults to "https://graphql.buildkite.com/v1", but can also be sourced from the `BUILDKITE_GRAPHQL_URL` environment variable.
-- `rest_url` - (Optional) This is the the base URL to use for REST requests. It defaults to "https://api.buildkite.com", but can also be sourced from the `BUILDKITE_REST_URL` environment variable.
+-   `api_token` - (Required) This is the Buildkite API Access Token. It must be provided but can also be sourced from the `BUILDKITE_API_TOKEN` environment variable.
+-   `organization` - (Required) This is the Buildkite organization slug. It must be provided, but can also be sourced from the `BUILDKITE_ORGANIZATIO_SLUG_` environment variable. The token requires GraphQL access and the `write_pipelines, read_pipelines` scopes.
```

Comment:
@james2791 I didn't see this. It'll have to be changed in the next merge. If you suggest changes then I can commit them in rather than a comment with approval 🙂

Reply:
Fair :)